contributing/webapp.md

```diff
@@ -182,12 +266,10 @@ And then chose "Import Module from external model" and take Maven
 ```
 <rudder's path directory>/webapp/sources/rudder/rudder-web/src/main/webapp
 ```
-
 - Classes Folder :
 ```
-<rudder's path directory>/webapp/sources/rudder/rudder-core/target/classes, <rudder's path directory>/webapp/sources/rudder/rudder-web/target/classes
+<rudder's path directory>/webapp/sources/rudder/rudder-web/target/classes
 ```
-
 - Runs on Port : 8080
 - VM Args :
 ```
```

Comment:
In the VM args `-Drudder.configFile=/home/elaad/rudder-tests/dev/configuration.properties` I think we should remove my path to avoid copy/paste error

Reply:
yes!